src/commands/protocols/publish/receiver/v1.0.0/v1-0-0-handle-store-request-command.js

```diff
@@ -52,7 +53,8 @@ class HandleStoreRequestCommand extends HandleProtocolMessageCommand {
         );
 
         const ual = this.ualService.deriveUAL(blockchain, contract, tokenId);
-        const assetExists = await this.tripleStoreModuleManager.assetExists(
+        const assetExists = await this.tripleStoreService.assetExists(
+            TRIPLE_STORE_REPOSITORIES.PRIVATE_CURRENT,
```

Comment:
Why are we here storing to private repository?

Reply:
This part is still work in progress. It will be updated soon